### PR TITLE
Release 2.0.0rc5

### PR DESCRIPTION
**Pull request by Hind-M**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Hind-M** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml
Comment (jjerphan) [inline recipe/meta.yaml:1]: We need to port the changes of https://github.com/conda-forge/micromamba-feedstock/pull/200 here.

**Review by jjerphan — APPROVED** [review]
LGTM.

Thank you, @Hind-M.

File: recipe/meta.yaml
Comment (Hind-M) [reply recipe/meta.yaml:1]: Yep

File: recipe/meta.yaml [hunk 1]
```diff
@@ -50,6 +50,7 @@ outputs:
         - zstd
         - simdjson >=3.3.0
       run:
+        - libcurl <8.10
```
Comment (jjerphan) [inline recipe/meta.yaml:53]: We are surprised that this is not handled upstream.
See: https://github.com/conda-forge/curl-feedstock/issues/143.

**Pull request merged**